Framework/script/RepoCleaner/repoCleaner.py

```diff
@@ -128,6 +129,25 @@ def downloadConfigFromGit():
     return path
 
 
+def downloadConfigFromConsul(consul_url, consul_port):
+    """
+    Download a config file from consul.
+    :return: the path to the config file
+    """
+
+    logging.debug("Get it from consul")
+    consul_server = consul.Consul(host=consul_url, port=consul_port)
+    index, data = consul_server.kv.get(key='folder/blabla')
+    logging.debug(f"config file from consul : \n{data['Value']}")
+    text = data["Value"].decode()
+    logging.debug(f"config file from consul : \n{text}")
```

Comment:
Just a question: how are the supposed to be different?

Reply:
the data returned by Consul is encoded in base64 for some reason.